lib/trashed/consts.rb

```diff
@@ -0,0 +1,3 @@
+module Trashed
+  STATE, COUNTERS, GAUGES = 'trashed.state', 'trashed.counters', 'trashed.gauges'
```

Comment:
This assignment can cause issues. If you accidentally add a value without a constant then you'll silently assign one of the constants as an array.

Under the hood multiple assignment also allocates an array, if you're doing it a lot it's a performance concern, here it's not a perf hotspot, but wanted to mention.